gotechnica/platform-2020
# Adds timestamps to teams, invites, and memberships

Description
---
Very short PR that adds timestamps when teams are created, invitations are sent, and when invitations are accepted.

Steps to Test
---
Use any of the `create_team`, `invite_to_team`, or `join_team` services and you will see an item added to the respective table with a timestamp.
